pom.xml

```diff
@@ -188,6 +188,11 @@
             <version>${httpcomponents.version}</version>
         </dependency>
         <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpasyncclient</artifactId>
+            <version>4.1.2</version>
```

Comment:
ahh I see it is unrelated. this might be obvious to those who know this library, but not intuitive. mind adding an XML comment that this has a different versioning scheme vs the other files?